common/SmokeTests/SmokeTest/Samples/IotHubConnection/IotHubConnectionTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SmokeTest.Samples
+{
+    public static class IotHubConnectionTests
+    {
+        public static async Task RunTests()
+        {
+            Console.WriteLine("---------------------------------------------");
```

Comment:
Same with this boiler plate code. Can we not use reflection to call the entry point for these samples?

Reply:
I'll defer to @danieljurek on that.   I certainly agree with the perspective, but the goal of creating a robust infrastructure for smoke tests orthogonal to my immediate goal of ensuring test coverage on this single sample.   I'm happy to help with the efforts, but I'll need to block additional time at a later point to do so.